packages/create-jazz-app/src/index.ts

```diff
@@ -369,7 +370,9 @@ module.exports = mergeConfig(getDefaultConfig(__dirname), config);`;
     // Clean up temp directory
     fs.rmSync(tempDocsDir, { recursive: true, force: true });
     // Fetch the latest llms-full.txt from the web
-    const response = await fetch("https://jazz.tools/llms-full.txt");
+    const response = await fetch(
+      `https://jazz.tools/${framework === "nextjs" ? "react" : framework}/llms-full.txt`,
+    );
```

Comment:
Would it be possible to add some tests that verify that thi always return an exisiting URL?